distribution/zip/jballerina-tools/build.gradle

```diff
@@ -119,6 +119,7 @@ dependencies {
     docerina project(':docerina-gradle-plugin')
     dist project(':ballerina-shell:shell-core')
     dist project(':ballerina-shell:shell-cli')
+    dist project(':compiler-plugins:pkg-analyzer')
```

Comment:
Shall we have a name that is self-explanatory? Suggestion: pkg-toml-semantic-analyzer

Reply:
Renamed module to `pkg-toml-semantic-analyzer`